lib/src/front_end/piece_factory.dart

```diff
@@ -427,6 +431,34 @@ mixin PieceFactory implements CommentWriter {
         isValueDelimited: rightHandSide.isDelimited));
   }
 
+  /// Writes the condition and updaters part of for a [ForParts] after the
```

Comment:
Nit: Something wrong/typo with the `part of for a` comment.

Reply:
Thanks, fixed.